tests/markov/msm/test_tram.py

```diff
@@ -3,10 +3,11 @@
 from tqdm import tqdm
 from flaky import flaky
 
-from deeptime.markov.msm import TRAM
+from deeptime.markov.msm.tram import TRAM, TRAMDataset
```

Comment:
perhaps it's consistency-wise better to elevate TRAMDataset into the `msm` package too, what do you say?